src/main/java/io/github/structgraph/Main.java

```diff
@@ -70,6 +68,21 @@ protected void typesProcessed(int typeCounter) {
         System.out.println("Done in "+secondsSince(start)+ "s.");
     }
 
+    private static File prepareDbDirectory(String arg) throws IOException {
+        File dbLocation = new File(arg);
+        FileUtils.cleanDirectory(dbLocation);
```

Comment:
This might be a little surprising for a user - deleting their database directory without warning.
If it is a database directory at all -- Just imagine they swap their artifact and database arguments ;)

Either a prompt, or command line argument (`--force`) would be needed to enable this feature.